### PR TITLE
Added Juno RPC Schema

### PR DESCRIPTION
**Pull request by Brivan-26**
Closes [Schema for custom RPC endpoints](https://github.com/NethermindEth/juno/issues/936)

### CODE REVIEW

File: juno_rpc_api.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                {
+                    "$ref": "#/components/errors/INTERNAL_ERROR"
+                }
+            ]
+        }
+    ],
+    "components": {
+        "contentDescriptors": {},
+        "schemas": {
+            "FELT": {
```
[the remark this answers is not in the document]
Comment (Brivan-26) [reply juno_rpc_api.json:68]: You mean the common schemas such as `FELT` and `TXN_HASH` should be referred directly to the Starknet RPC spec ?

[the remark this answers is not in the document]
Comment (joshklop) [reply juno_rpc_api.json:68]: Preferably, yes.

[the remark this answers is not in the document]
Comment (Brivan-26) [reply juno_rpc_api.json:68]: They don't refer to the Starknet RPC space. I see it better to not, so who ever is using the spec doesn't need to switch between multiple standards, but find all components in the same place. What do you think?

[the remark this answers is not in the document]
Comment (joshklop) [reply juno_rpc_api.json:68]: If Pathfinder doesn't then let's not worry about it.

File: juno_rpc_api.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                "schema": {
+                    "type": "string",
+                    "description": "A semver-compatible version string"
+                }
+            }
+        },
+        {
+            "name": "juno_getTransactionStatus",
+            "summary": "Returns the status of a transaction",
+            "description": "This method returns the current status of a transaction. The status is comprised of both finality and execution statuses",
```
Comment (joshklop) [inline juno_rpc_api.json:25]: Can we remove the description or summary? Seems redundant to have both.

**Review by joshklop — CHANGES_REQUESTED** [review]

File: juno_rpc_api.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            },
+            "TXN_HASH": {
+                "$ref": "#/components/schemas/FELT",
+                "description": "The transaction hash, as assigned in Starknet",
+                "title": "A transaction's hash"
+            },
+            "TX_FINALITY_STATUS": {
+                "type": "string",
+                "enum": [
+                    "1",
```
Comment (joshklop) [inline juno_rpc_api.json:82]: The strings should be `"ACCEPTED_ON_L1"`, etc.

**Review by joshklop — APPROVED** [review]

**Pull request merged**